.ci/containers/terraform-tester/test_terraform.sh

```diff
@@ -29,7 +29,7 @@ pushd $local_path
 echo "Checking for modified go files"
```

Comment:
We can additionally remove the files from the exclusion list under https://github.com/GoogleCloudPlatform/magic-modules/blob/master/.ci/containers/downstream-builder/generate_downstream.sh#L95